packages/api-explorer/src/reducers/spec/utils.ts

```diff
@@ -173,6 +173,27 @@ export const getSpecsFromVersions = async (
   fetcher: SpecFetcher | undefined = undefined
 ): Promise<SpecItems> => {
   const items = {}
+
+  /**
+   * Create a unique spec key for this version
+   * @param v version to identify
+   */
+  const uniqueId = (v: IApiVersionElement) => {
```

Comment:
Is there a unit test for this? If not, could we add one as part of this PR please?